platform/client/.graphqlrc.yml

```diff
@@ -15,6 +15,9 @@ extensions:
       subUpdate: false
       subDelete: false
     plugins:
+      graphback-schema:
+        format: graphql
+        outputPath: ../server/src/schema
```

Comment:
With this I am loosing the `Subscription` from generated schema - see https://github.com/aerogear/OpenVolunteerPlatform/pull/56/files#diff-34468dd674ec64aa30c9baa2b7192a64L115-R168 

/cc @wtrocki

Reply:
Will be fixed by changing the outputPath. I'll use `./build` so that the generated schema file does not get committed.